app/hotels/README.md

```diff
@@ -21,6 +21,8 @@ import {
 <AllHotelsMap />
 ```
 
+Note: Currently you need to set Google Maps Api key manually to make it work on Android (see app.json), follow instructions in Expo [MapView Docs](https://docs.expo.io/versions/latest/sdk/map-view.html#deploying-to-a-standalone-app-on-android)
```

Comment:
How does it work if I want to use this component outside of our project? Because native developer will not have `app.json`...